exercises/concept/treasure-factory/.docs/instructions.md

```diff
@@ -29,12 +29,14 @@ A `Chest` should hold the same data as a `TreasureChest` and should have two typ
 Note that because `Chest` is using a phantom type, and should therefore be opaque to prevent its use anywhere outside of the `TreasureFactory` module.
 In this case, it is not even exposed at all.
 
-`secureChest` should take a `Chest` without specific conditions and return one wrapped in a `Maybe` that has the extra condition `securePassword : ()`.
-`makeTreasureChest` should take a `List` of `Chest` without specific conditions and return a `List` of `Chest`, with the duplicates removed, and the extra condition `uniqueTreasure : ()` added.
+Edit the type signatures of `secureChest` and `uniqueTreasures` to add the constraints using phantom types.
```

Comment:
Could we be a tiny bit more specific by saying "using extensible records as phantom types"?